README.md

```diff
@@ -127,7 +119,7 @@ and GoTestWAF evaluating ModSecurity. There are two options to run the demo envi
 You can also run NGINX‑based ModSecurity using OWASP Core Rule Set and GoTestWAF evaluating ModSecurity by using
 the `make` utility as follows (executed commands are defined in the Makefile located in the repository root):
 
-1. Clone this repository and go to the local folder:
+1. Clone this repository, go to the local folder, and run the following commands:
 
     ```
     git clone https://github.com/wallarm/gotestwaf.git
```

Comment:
The commands provided below do the following: clone the repo and open the cloned folder. The phrase "and run the following commands" changes the meaning

Reply:
Yes, you are right. Maybe it's better to add just "Clone this repository and go to the local directory"